src/pages/Login/login.styles.js

```diff
@@ -66,6 +66,37 @@ export const InputWrapper = styled.div`
   margin-bottom: 1.25rem;
 `;
 
+export const InputGroup = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+`;
+
+export const PasswordInputRapper = styled.div`
+  width: 100%;
+`;
+
+export const EyeIconRapper = styled.div`
```

Comment:
pretty sure I saw this around the app
```suggestion
export const IconContainer = styled.div`
```